dokka-integration-tests/gradle/src/main/kotlin/org/jetbrains/dokka/it/gradle/AbstractGradleIntegrationTest.kt

```diff
@@ -39,6 +39,24 @@ abstract class AbstractGradleIntegrationTest : AbstractIntegrationTest() {
         vararg arguments: String,
         jvmArgs: List<String> = listOf("-Xmx2G", "-XX:MaxMetaspaceSize=1G")
     ): GradleRunner {
+
+        // TODO quick hack to add `android { namespace }` on AGP 7+ (it's mandatory in 8+).
```

Comment:
can't we just set `namespace` in project?
I see that the property is available from `4.2.0` - https://developer.android.com/reference/tools/gradle-api/8.3/com/android/build/api/dsl/CommonExtension#namespace()

Reply:
Good idea. I did try setting it in all projects, but it failed. I didn't look too much into it, I just added this quick fix to get all the tests working locally.

Reply:
The problem is, that for some reason we execute integration tests for Android with AGP 4.0.0 (and it even works) - I believe we could simplify this in following PR by testing with AGP 7+